public/app/project-setup/theme/variables/_layout.scss

```diff
@@ -1 +1,2 @@
-$global-padding: 50px;
+$basic-padding: 50px;
```

Comment:
`basic-padding` _is_ `column_width`. Why not making it explicit?

`basic-padding: $column-width`